src/index.scss

```diff
@@ -93,38 +93,42 @@ $white: #fff;
     .nav-link:focus,
     .nav-link.active,
     .expanded .nav-link {
-      background: $component-active-bg;
-      color: $component-active-color;
+      background: var(--pgn-color-bg-active, $component-active-bg);
+      color: var(--pgn-color-active, $component-active-color);
     }
   }
   .main-nav {
     .nav-link {
-      padding: 1.125rem 1rem;
+      padding: 1.125rem var(--pgn-spacing-spacer-base, $spacer);
       text-decoration: none;
       font-weight: 500;
       letter-spacing: .01em;
     }
+
     .nav-link:hover,
     .nav-link:focus,
     .nav-link.active,
     .expanded .nav-link {
-      background: $component-active-bg;
-      color: $component-active-color;
+      background: var(--pgn-color-bg-active, $component-active-bg);
+      color: var(--pgn-color-active, $component-active-color);
     }
+
     .menu {
       position: static;
+
       .menu-content {
-        border-top: solid 2px $component-active-bg;
```

Comment:
> [!NOTE]
> To keep this compatible we need import `core.scss` into the MFEs.

Reply:
That is a note applicable to the new version (Paragon 23)  taking in mind that the migration [guide](https://openedx.atlassian.net/wiki/spaces/BPL/pages/3770744958/Migrating+MFEs+to+Paragon+design+tokens+and+CSS+variables#Remove-all-(almost)-Paragon-imports-from-the-SCSS-entry-point-in-your-MFE) said:

> The only import from Paragon needed now is for its custom-media-breakpoints. This must be imported within your MFE in order to use the CSS media queries using Paragon’s defined breakpoints.

Reply:
Ah! Got it. Could we work around that by adding definitions the scss vars we use in this file *to* this file but using `!default`?

Reply:
I made some changes to define the defaults, could you have a look, please?

Reply:
The defaults look great!